planning/behavior_path_planner/src/scene_module/lane_change/interface.cpp

```diff
@@ -207,7 +207,12 @@ BehaviorModuleOutput LaneChangeInterface::plan()
   auto output = module_type_->generateOutput();
   path_reference_ = output.reference_path;
   *prev_approved_path_ = *getPreviousModuleOutput().path;
-  module_type_->insertStopPoint(module_type_->getLaneChangeStatus().target_lanes, *output.path);
+
+  if (module_type_->isAbortState()) {
+    module_type_->insertStopPoint(module_type_->getLaneChangeStatus().current_lanes, *output.path);
+  } else {
+    module_type_->insertStopPoint(module_type_->getLaneChangeStatus().target_lanes, *output.path);
+  }
```

Comment:
@rej55 
This part should be in the `generateOutput()` function, as it is related to implementation and not interface to scene module.

Reply:
Done https://github.com/autowarefoundation/autoware.universe/pull/5509/commits/b68837f4f90423656405db9e8371b57c0e01a19a